acvm-repo/acvm_js/Cargo.toml

```diff
@@ -41,5 +41,5 @@ wasm-bindgen-test.workspace = true
 
 [features]
 default = ["bn254"]
-bn254 = ["acvm/bn254", "barretenberg_blackbox_solver/bn254"]
-bls12_381 = ["acvm/bls12_381", "barretenberg_blackbox_solver/bls12_381"]
+bn254 = ["acvm/bn254", "bn254_blackbox_solver/bn254"]
+bls12_381 = ["acvm/bls12_381", "bn254_blackbox_solver/bls12_381"]
```

Comment:
> bn254_blackbox_solver/bls12_381

This doesn't really make much sense anymore

Reply:
ah, whoops